spec/integration/css_helper_spec.rb

```diff
@@ -39,6 +39,15 @@ def expect_file(path, content='file content')
         expect(css_for_doc(doc)).to eq("content of base.css\ncontent of font.css")
       end
     end
+
+    context 'when HTML contains ignored links' do
+      let(:files) { ['ignore.css', {'data-premailer' => 'ignore'}] }
```

Comment:
Could you add spacing around the curly braces here? Alternatively, this would also work without curly braces since it's the last argument :blush: